src/core/ConfigManager.cpp

```diff
@@ -253,16 +254,17 @@ void ConfigManager::createWorkingDir()
 
 
 
-void ConfigManager::addRecentlyOpenedProject( const QString & _file )
+void ConfigManager::addRecentlyOpenedProject( const QString & file )
 {
-	if( !_file.endsWith( ".mpt", Qt::CaseInsensitive ) ) 
+	QFileInfo recentFile( file );
+	if( recentFile.suffix() == "mmp" || recentFile.suffix() == "mmpz" ) 
```

Comment:
Previous logic treated the suffix as case-insensitive. It may be a good idea to change this to
```
if( recentFile.suffix().toLower() == "mmp" || recentFile.suffix().toLower() == "mmpz" )
```

In order to match this behavior.